auto_round_extension/qbits/qlinear_qbits.py

```diff
@@ -20,12 +20,6 @@
 import torch.nn as nn
 from auto_round.utils import convert_dtype_torch2str, logger
 QBITS_AVAILABLE = True
-try:
```

Comment:
this change will lead qbits.repack_quantized_weight, qbits.woq_linear throw `no attribute` err.
the solution is to add a member named self.qbits, in post_init stage, pls set `self.qbits=intel_extension_for_transformers.qbits`

don't forget to import itrex when we try to get itrex-version in req_check.

i have validated that autoround works well after updating these changes.